server/src/main/java/org/opensearch/cluster/service/ClusterManagerTaskThrottler.java

```diff
@@ -50,6 +51,10 @@ public class ClusterManagerTaskThrottler implements TaskBatcherListener {
     private final ConcurrentMap<String, Long> tasksCount;
     private final ConcurrentMap<String, Long> tasksThreshold;
     private final Supplier<Version> minNodeVersionSupplier;
+    // Needed for static threshold settings.
+    // Once all nodes are higher than 2.5.0 version, then only it will start throttling.
```

Comment:
>= 2.5.0

Reply:
Ack

Reply:
Once all nodes are higher than 2.5.0 version, then only it will start throttling. During upgrade as well, it will wait for all older version nodes to leave the cluster and then only start throttling. This is needed specifically for static setting to enable throttling.

Reply:
Thanks for catching that. Corrected it.